### ISSUE TITLE
termencoding was removed from Neovim

### ISSUE THREAD

**Issue by taq** [majutsushi/tagbar#577]
Seems that `termencoding` was removed from Neovim. From it's help page:

```
==============================================================================                                                                  
6. Removed features                                      nvim-features-removed                                                                  
                                                                                                                                                
These Vim features were intentionally removed from Nvim.     
...
Options:                                                                                                                                        
  'antialias'       
...
  'termencoding' 'tenc' (Vim 7.4.852 also removed this for Windows) 
```
As this plugin is using `termencoding`, I have now some issues when opening Neovim.

**Comment by alerque** [comment]
Thanks for reporting this. It was also reported yesterday in #574 and fixed in #575. You should be able to update your plugins ta have the error produced by the use of this fixed.

**Comment by taq** [comment]
Thank you so much!
